### DISCUSSION TITLE
Could not update the allowed domain for my Dapp

### DISCUSSION THREAD

**Discussion by natsabari** [WalletConnect/walletconnect-monorepo#3102] [Developer Support]
We have wallet connect implemented for [chia ](https://www.chia.net/) in https://spacescan.io and wanted to update the allowed domains .
when ever i tried to update it its giving me below error.

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2860620/cd86d263-798a-4b97-a7ce-00b0c10bb196)

**Comment by boidushya** [comment]
Hey, this might've been caused by a network issue. Tried replicating this but seems like its working now.
Could you try again and let us know if its fixed for you?

**Reply by natsabari** [reply]
I tried it again now and getting same issue 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2860620/576f152d-189b-44df-8f4c-0fa74daa20c4)

**Comment by natsabari** [comment]
Team - Any update on this issue, we have released this wallet connect feature but its throwing below error because our naked domain not in that list.

https://spacescan.io -Throwing below error . https://www.spacescan.io - working fine

{context: 'core/relayer'} Error: WebSocket connection closed abnormally with code: 3000 (Authorization error: Unauthorized: origin not allowed)

**Reply by natsabari** [reply]
@boidushya  Could you please help, we are stuck with this issue

**Reply by natsabari** [reply]
Verifying the domain is tied to the explorer submission but explorer submission is failing with similar fetch error!!
Please fix those bugs and its really annoying and blocking us from enabling wallet connect !!

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2860620/74bd50f2-371c-407d-8ad4-b0e31fc7e1c3)

**Comment by natsabari** [comment]
Working now and closing this
